### DISCUSSION TITLE
About the cluster deployment solution for Milvus 1.x version

### DISCUSSION THREAD

**Discussion by ingale726** [milvus-io/milvus#22119] [Q&A and General discussion]
Will Milvus 1.x cluster?There is not much documentation on this. Can you provide me with some relevant documents?

**Comment by yhmo** [comment]
There is a middleware written by python to create v1.x cluster: 
https://github.com/milvus-io/milvus/tree/1.x/shards

Note: 
this middleware still has some unsolved problems:
1. cannot ensure data consistency, some newly inserted rows might be unvisible in a short time
2. scale out query nodes, then shut down some query nodes, the cluster will not be able to do search
